megalinter/descriptors/salesforce.megalinter-descriptor.yml

```
@@ -227,37 +229,40 @@ linters:
     cli_config_arg_name: ".eslintrc-lwc.json"
     cli_executable: sf
     cli_lint_extra_args:
-      - "scanner:run"
-      - "--engine"
-      - "eslint-lwc"
+      - "code-analyzer"
+      - "run"
+      - "--rule-selector"
+      - "eslint:LWC"
+      - "--workspace"
+      - "."
       - "--target"
       - "**/lwc/**/*.js"
       - "--severity-threshold"
       - "2"
-      - "--outfile"
-      - "{{REPORT_FOLDER}}/sfdx-scanner-report-lwc.csv"
-      - "--format"
-      - "csv"
+      - "--output-file"
+      - "{{REPORT_FOLDER}}/code-analyzer-report-lwc.csv"
+      - "--view"
+      - "table"
     cli_lint_mode: project
     cli_help_extra_args:
-      - "scanner:run"
+      - "code-analyzer run"
     cli_help_extra_commands:
-      - "sf scanner:rule:list"
+      - "sf code-analyzer rules --rule-selector eslint:LWC"
     cli_version_extra_args:
-      - "plugins"
+      - "plugins --core"
     cli_version_arg_name: ""
-    version_extract_regex: "(?<=@salesforce/sfdx-scanner )\\d+(\\.\\d+)+"
+    version_extract_regex: "(?<=code-analyzer )\\d+(\\.\\d+)+"
     cli_lint_errors_count: regex_sum
-    cli_lint_errors_regex: "found ([0-9]+) violation"
+    cli_lint_errors_regex: "severity: [1-3]"
     examples:
-      - "sf scanner:run"
+      - "sf code-analyzer run --rule-selector eslint:LWC --workspace . --target '**/lwc/**/*.js' --output-file report.csv"
     install:
       dockerfile:
         - |-
-          # renovate: datasource=npm depName=@salesforce/sfdx-scanner
-          ARG SALESFORCE_SFDX_SCANNER_VERSION=4.12.0
+          # renovate: datasource=npm depName=@salesforce/code-analyzer
```

```suggestion
          # renovate: datasource=npm depName= @salesforce/plugin-code-analyzer
```

And then run the build script, or change everywhere